### PR TITLE
Remove `types` Argument

### PR DESCRIPTION
**Pull request by SudharakaP**
⚠️ Should be merged after https://github.com/opencollective/opencollective-api/pull/6344 and https://github.com/opencollective/opencollective-frontend/pull/6679

Related to https://github.com/opencollective/opencollective/issues/4378

### CODE REVIEW

File: server/graphql/v2/interface/Account.js [hunk 1]
```diff
@@ -265,7 +265,7 @@ const accountFieldsDefinition = () => ({
     type: new GraphQLList(PaymentMethod),
     description: 'The list of payment methods that this account can use to pay for Orders',
     args: {
-      types: {
+      type: {
```
Comment (Betree) [inline server/graphql/v2/interface/Account.js:268]: Looks like we should remove this block entirely

**Review by Betree — COMMENTED** [review]
The code should be fixed (see lint issue), but I have no blocker with the rest

File: server/graphql/v2/interface/Account.js (hunk 1 above)
Comment (SudharakaP) [reply server/graphql/v2/interface/Account.js:268]: Ops, this is my fault; I did some mistake in the rebase 🤯 😑

**Comment by SudharakaP** [comment]
@Betree: Corrected everything, if no objections I'll merge this one. 😄

File: server/graphql/v1/CollectiveInterface.js [hunk 2]
```diff
@@ -1776,8 +1770,8 @@ const CollectiveFields = () => {
         if (args.service) {
           paymentMethods = paymentMethods.filter(pm => pm.service === args.service);
         }
-        if (args.types) {
-          paymentMethods = paymentMethods.filter(pm => args.types.includes(pm.type));
+        if (args.type) {
+          paymentMethods = paymentMethods.filter(pm => args.type.includes(pm.type));
```
Comment (Betree) [inline server/graphql/v1/CollectiveInterface.js:1774]: Similar feedback here, it seems to duplicate the two line below

Comment (SudharakaP) [reply server/graphql/v1/CollectiveInterface.js]: Ops, corrected that one as well. Thanks for catching that. 🙂

**Comment by znarf** [comment]
@SudharakaP can you rebase this PR so we trigger the test again?

**Comment by SudharakaP** [comment]
> @SudharakaP can you rebase this PR so we trigger the test again?

Done. Let us see if the tests pass. Let me know if there's anything to do here. 👍🏽

**Review by znarf — APPROVED** [review]

**Pull request merged**